ShipEngine/Models/Dto/GetRatesFromShipmentDetails/Params.cs

```diff
@@ -145,6 +145,10 @@ public class Shipment
         /// The combined weight of all packages in the shipment
         /// </summary>
         public Weight Weight { get; set; }
+
+        [JsonConverter(typeof(StringEnumConverter))]
+        public ComparisonRateType? ComparisonRateType { get; set; }
```

Comment:
Do you know what happens if a different ComparisonRateType is returned that isn't in the ComparisonRateType enum?

Reply:
This is the request, not the response.